invenio_records_resources/resources/record.py

```diff
@@ -46,22 +69,21 @@ def __init__(
     ):
         """Constructor."""
         super(RecordResource, self).__init__(*args, **kwargs)
-        self.service_cls = service_cls
```

Comment:
Moderate: Was this supposed to be removed? It's being used below...

Reply:
yes sorry I added it back :)